README.md

```diff
@@ -124,5 +124,10 @@ and then use `cargo` to launch the tool :
 ```bash
 $ cargo run -- <ssc_options>
 ```
-
+## Contribution guide
 Also, see our [contributing guide](CONTRIBUTING.md).
+
+## Maintainers 
+[nm2107](https://github.com/nm2107)
+[neghmurken](https://github.com/neghmurken)
+[jaljo](https://github.com/jaljo)
```

Comment:
:thinking: 
For me you can keep only Nico in maintainers.

Reply:
@neghmurken , @jaljo , do you accept to be listed as maintainers ?

I'll cherry pick the commit to rebase against master once I've got your answers.

Reply:
Sorry, I'll politely decline. I have absolutely no knowledge of this project or of Rust whatsoever

Reply:
No problem, thank you for answering :)

@jaljo, what about you ? (no obligations there :) )